src/SortAlgorithms/index.php

```diff
@@ -0,0 +1,15 @@
+<?php
```

Comment:
It would be nice to have comparison between both algorithms on amount of iterations.